store/s3.go

```diff
@@ -16,10 +19,9 @@ import (
 var _ Storage = (*AWSS3Storage)(nil)
 
 type AWSS3Storage struct {
-	bucket          string
-	region          string
-	accessKeyID     string
-	secretAccessKey string
+	bucket string
+	region string
+	sess   *session.Session
```

Comment:
store `*session.Session` of s3 storage when creating new one for reusing the session in download.